painters/train_cnn.py

```diff
@@ -161,7 +184,9 @@ def load_trained_cnn_softmax_layer(model_path):
 
 
 def _load_trained_cnn_layer(model_path, layer_index):
-    model = load_model(model_path)
+    model = model_from_json('../models/model.json')
+    model.load_weights('../models/weights.h5')
```

Comment:
Is it still necessary to hardcode these paths? If so, consider removing the `model_path` argument from the function as it is no longer used.